python/ray/client_builder.py

```diff
@@ -229,8 +187,6 @@ def _get_builder_from_address(address: Optional[str]) -> ClientBuilder:
             pass
         return _LocalClientBuilder(address)
```

Comment:
is this needed?

Reply:
Yup for backwards compat, 176 is for ray.client("local").connect(), 188 is for ray.client().connect()